src/card/aggregate-status/aggregate-status-card.component.js

```diff
@@ -68,7 +68,8 @@
  </file>
 
  <file name="script.js">
-   angular.module( 'patternfly.card' ).controller( 'CardDemoCtrl', function( $scope ) {
+   angular.module( 'patternfly.card' ).controller( 'CardDemoCtrl', function( $scope, $window ) {
+    var imagePath = $window.IMAGE_PATH || "img";
```

Comment:
we should be consistent and use /img

Reply:
I was trying to preserve what was previously present.  Would you say `img` is expected to be correct over '/img'

Reply:
yeah, I would say 'img' is probably better, I'd just rather see consistency.

Reply:
Done.  Standardised on `img`.